org.adempiere.base.process/src/org/compiere/process/ImportGLJournal.java

```diff
@@ -209,85 +209,85 @@ protected String doIt() throws java.lang.Exception
-			.append("WHERE (C_Currency_ID IS NULL OR C_Currency_ID=0)")
+			.append(" SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=Invalid Currency, '")
+			.append(" WHERE (C_Currency_ID IS NULL OR C_Currency_ID=0)")
 			.append(" AND I_IsImported<>'Y'").append (clientCheck);
 		no = DB.executeUpdate(sql.toString(), get_TrxName());
 		if (no != 0)
 			log.warning ("Invalid Currency=" + no);
 
 		//	Set Conversion Type
 		sql = new StringBuilder ("UPDATE I_GLJournal i ") 
-			.append("SET ConversionTypeValue='S' ")
-			.append("WHERE C_ConversionType_ID IS NULL AND ConversionTypeValue IS NULL")
+			.append(" SET ConversionTypeValue='S' ")
+			.append(" WHERE C_ConversionType_ID IS NULL AND ConversionTypeValue IS NULL")
 			.append(" AND I_IsImported='N'").append (clientCheck);
 		no = DB.executeUpdate(sql.toString(), get_TrxName());
 		if (log.isLoggable(Level.FINE)) log.fine("Set CurrencyType Value to Spot =" + no);
 		sql = new StringBuilder ("UPDATE I_GLJournal i ") 
-			.append("SET C_ConversionType_ID=(SELECT c.C_ConversionType_ID FROM C_ConversionType c")
+			.append(" SET C_ConversionType_ID=(SELECT c.C_ConversionType_ID FROM C_ConversionType c")
 			.append(" WHERE c.Value=i.ConversionTypeValue AND c.AD_Client_ID IN (0,i.AD_Client_ID)) ")
-			.append("WHERE C_ConversionType_ID IS NULL AND ConversionTypeValue IS NOT NULL")
+			.append(" WHERE C_ConversionType_ID IS NULL AND ConversionTypeValue IS NOT NULL")
 			.append(" AND I_IsImported<>'Y'").append (clientCheck);
 		no = DB.executeUpdate(sql.toString(), get_TrxName());
 		if (log.isLoggable(Level.FINE)) log.fine("Set CurrencyType from Value=" + no);
 		sql = new StringBuilder ("UPDATE I_GLJournal i ")
-			.append("SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=Invalid CurrencyType, '")
-			.append("WHERE (C_ConversionType_ID IS NULL OR C_ConversionType_ID=0) AND ConversionTypeValue IS NOT NULL")
+			.append(" SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=Invalid CurrencyType, '")
+			.append(" WHERE (C_ConversionType_ID IS NULL OR C_ConversionType_ID=0) AND ConversionTypeValue IS NOT NULL")
 			.append(" AND I_IsImported<>'Y'").append (clientCheck);
 		no = DB.executeUpdate(sql.toString(), get_TrxName());
 		if (no != 0)
 			log.warning ("Invalid CurrencyTypeValue=" + no);
 
 
 		sql = new StringBuilder ("UPDATE I_GLJournal i ")
-			.append("SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=No ConversionType, '")
-			.append("WHERE (C_ConversionType_ID IS NULL OR C_ConversionType_ID=0)")
+			.append(" SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=No ConversionType, '")
+			.append(" WHERE (C_ConversionType_ID IS NULL OR C_ConversionType_ID=0)")
 			.append(" AND I_IsImported<>'Y'").append (clientCheck);
 		no = DB.executeUpdate(sql.toString(), get_TrxName());
 		if (no != 0)
 			log.warning ("No CourrencyType=" + no);
 
 		//	Set/Overwrite Home Currency Rate
 		sql = new StringBuilder ("UPDATE I_GLJournal i ")
-			.append("SET CurrencyRate=1")

```

Comment:
Note that issue was at this line. but updated all lines to start with space to make sql auditable.